branca/colormap.py

```diff
@@ -9,223 +9,25 @@
-            'RdPu': ['#FEEBE2', '#FCC5C0', '#FA9FB5',
-                     '#F768A1', '#DD3497', '#AE017E', '#7A0177'],
-            'YlGn': ['#FFFFCC', '#D9F0A3', '#ADDD8E',
-                     '#78C679', '#41AB5D', '#238443', '#005A32'],
-            'YlGnBu': ['#FFFFCC', '#C7E9B4', '#7FCDBB',
-                       '#41B6C4', '#1D91C0', '#225EA8', '#0C2C84'],
-            'YlOrBr': ['#FFFFD4', '#FEE391', '#FEC44F',
-                       '#FE9929', '#EC7014', '#CC4C02', '#8C2D04'],
-            'YlOrRd': ['#FFFFB2', '#FED976', '#FEB24C',
-                       '#FD8D3C', '#FC4E2A', '#E31A1C', '#B10026'],
-            'BrBg': ['#8c510a', '#d8b365', '#f6e8c3',
-                     '#c7eae5', '#5ab4ac', '#01665e'],
-            'PiYG': ['#c51b7d', '#e9a3c9', '#fde0ef',
-                     '#e6f5d0', '#a1d76a', '#4d9221'],
-            'PRGn': ['#762a83', '#af8dc3', '#e7d4e8',
-                     '#d9f0d3', '#7fbf7b', '#1b7837'],
-            'PuOr': ['#b35806', '#f1a340', '#fee0b6',
-                     '#d8daeb', '#998ec3', '#542788'],
-            'RdBu': ['#b2182b', '#ef8a62', '#fddbc7',
-                     '#d1e5f0', '#67a9cf', '#2166ac'],
-            'RdGy': ['#b2182b', '#ef8a62', '#fddbc7',
-                     '#e0e0e0', '#999999', '#4d4d4d'],
-            'RdYlBu': ['#d73027', '#fc8d59', '#fee090',
-                       '#e0f3f8', '#91bfdb', '#4575b4'],
-            'RdYlGn': ['#d73027', '#fc8d59', '#fee08b',
-                       '#d9ef8b', '#91cf60', '#1a9850'],
-            'Spectral': ['#d53e4f', '#fc8d59', '#fee08b',
-                         '#e6f598', '#99d594', '#3288bd'],
-            'Accent': ['#7fc97f', '#beaed4', '#fdc086',
-                       '#ffff99', '#386cb0', '#f0027f'],
-            'Dark2': ['#1b9e77', '#d95f02', '#7570b3',
-                      '#e7298a', '#66a61e', '#e6ab02'],
-            'Paired': ['#a6cee3', '#1f78b4', '#b2df8a',
-                       '#33a02c', '#fb9a99', '#e31a1c'],
-            'Pastel1': ['#fbb4ae', '#b3cde3', '#ccebc5',
-                        '#decbe4', '#fed9a6', '#ffffcc'],
-            'Pastel2': ['#b3e2cd', '#fdcdac', '#cbd5e8',
-                        '#f4cae4', '#e6f5c9', '#fff2ae'],
-            'Set1': ['#e41a1c', '#377eb8', '#4daf4a',
-                     '#984ea3', '#ff7f00', '#ffff33'],
-            'Set2': ['#66c2a5', '#fc8d62', '#8da0cb',
-                     '#e78ac3', '#a6d854', '#ffd92f'],
-            'Set3': ['#8dd3c7', '#ffffb3', '#bebada',
-                     '#fb8072', '#80b1d3', '#fdb462']}
+
+resource_package = __name__
+resource_path_schemes = '/_schemes.json'
+resource_path_cnames = '/_cnames.json'
+
+cnames_string = pkg_resources.resource_stream(
+    resource_package, resource_path_cnames).read().decode()
+_cnames = json.loads(cnames_string)
+
+schemes_string = pkg_resources.resource_stream(
+    resource_package, resource_path_schemes).read().decode()
+_schemes = json.loads(schemes_string)
```

Comment:
We can probably ditch package `pkg_resources` and use the `os` module like:

```python
rootpath = os.path.abspath(os.path.dirname(__file__))
resource_path_schemes = os.path.join(rootpath, '_schemes.json')
```

Reply:
I'll leave it up to you, just let me know. I can re-write for os.path; I just thought pkg_resources was cleaner. I'm not sure either way though if one approach or the other is best practice.